docs/en/sql-reference/sql-functions/table-functions/files.md

```diff
@@ -94,7 +95,79 @@ The URI used to access the files. You can specify a path or a file.
 
 ### data_format
 
-The format of the data file. Valid values: `parquet` and `orc`.
+The format of the data file. Valid values: `parquet`, `orc`, and `csv`.
+
+You must set detailed options for specific data file formats.
+
+#### CSV
+
+Example for the CSV format:
+
+```SQL
+"format"="csv",
+"csv.column_separator"="\\t",
+"csv.enclose"='"',
+"csv.skip_header"="1",
+"csv.escape"="\\"
+```
+
+##### csv.column_separator
+
+Specifies the column separator used  when the data file is in CSV format. By default, if you do not specify this parameter, this parameter defaults to `\\t`, indicating tab. The column separator you specify using this parameter must be the same as the column separator that is actually used in the data file. Otherwise, the load job will fail due to inadequate data quality.
```

Comment:
1. extra space between "used  when"
2. "By default" can be deleted because we have "this parameter defaults to"